test/testdata/compiler/cross_file_compiled_final_method__2.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+# typed: true
+# compiled: true
+
+def f
+  Utils.final_method
```

Comment:
Is it worth adding some filecheck annotations here?

Reply:
We have filechecks in `call_final.rb` and other places; I'm not sure what additional ones we would have here.